src/app/globals.css

```
@@ -292,3 +299,160 @@ td {
+.systemd-boot-cursor-visible {
+  opacity: 1;
+}
+
+.systemd-boot-skip-sr {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)